### ISSUE TITLE
Export all calendars as a zip file

### ISSUE THREAD

**Issue by tcitworld** [nextcloud/calendar#257]
If you have multiple calendars, you don't want to export each of them when migrating to another instance, so lets provide : 
* a server-side route to loop on all your .ics files and add them to the zip file
* a button in the settings area to access the export path

**Comment by tcitworld** [comment]
It can concern only personal calendars for starters

**Comment by tcitworld** [comment]
This looks like it will require server changes, as there's no public API for calendars. https://github.com/nextcloud/calendar/issues/257

**Comment by georgehrke** [comment]
@tcitworld Did you see this yet? https://github.com/nextcloud/server/issues/6773

Maybe it's a good alternative to the zip archive approach.

**Comment by tcitworld** [comment]
In regard to above, I'm guessing it should be done after #878 then.

**Comment by georgehrke** [comment]
Closing as Duplicate of nextcloud/server#2864
